### PR TITLE
WIP: Fix/join event bugs

### PR DESCRIPTION
**Pull request by lmontopo**
* Fix spamming triggered by changing date command - eventName was being passed into job commands instead of event object, hence the date was undefined. 
* Fix the date of new events 
* Update errors in the command descriptions
* Remove tests 
* Fix shaming behaviour

### CODE REVIEW

File: scripts/socialBot.coffee [hunk 1]
```diff
@@ -580,6 +581,9 @@ test = (res) ->
     }
   })
 
+  console.log(schedule.scheduledJobs)
```
Comment (mahimabhayana) [inline scripts/socialBot.coffee]: remove console log :)

**Comment by mahimabhayana** [comment]
looks good! one thing I'd suggest while you're at it is to change the 'i'm in' command. Right now, help tells the user to use the command "I'm in for" vs actual behaviour which expects only "i'm in".

File: scripts/socialBot.coffee [hunk 2]
```diff
@@ -60,7 +60,7 @@ NO_ONE_VOTED = (eventName, creators) -> "#{creators} No one voted in the poll fo
 TIME_CHANGE_DURING_POLL_FORBIDDEN = (user, eventName) -> "@#{user} Wait for poll to end before changing the time for #{eventName}."
 NOT_CREATOR = (user, creators, eventName) -> "@#{user} Only #{creators} can modify #{eventName}."
 CREATOR_BREAK_TIE = (eventName, creators, winners) -> "#{creators} The poll for #{eventName} resulted in a tie. Tag socialbot with one of the winners:\n#{winners}"
-EVENT_FOLLOW_UP = (eventName, creators) -> "#{creators} Who showed up for the <event-name> yesterday?"
+EVENT_FOLLOW_UP = (eventName, creators) -> "#{creators} Who showed up for the #{eventName}?"
```
Comment (bellkelly) [inline scripts/socialBot.coffee]: Maybe remove 'the'? Or add 'event' to the end of the sentence?

`Who showed up for #{eventName}?`
or
`Who showed up for the #{eventName} event?`

Comment (mahimabhayana) [reply scripts/socialBot.coffee]: good catch! fixed :)

File: scripts/socialBot.coffee [hunk 3]
```diff
@@ -551,34 +552,24 @@ vote = (res) ->
...
-test = (res) ->
-  parseOptions({
-    eventName: 'Kelly',
-    options: {
-      'today': 0,
-      'tmr': 0
-    }
-  })
+  if bailers.length > 0
+    res.send SHAME(eventName, parseNotify(bailers))
```
Comment (bellkelly) [inline scripts/socialBot.coffee]: I really love that shaming was added! haha!

**Pull request merged**